### ISSUE TITLE
Mask-RCNN tutorial one-line change suggestion

### ISSUE THREAD

**Issue by morawi** [pytorch/tutorials#960]
I would like to suggest changing the labels in [Mask-RCNN tutorial](https://pytorch.org/tutorials/intermediate/torchvision_tutorial.html) from 

`labels = torch.ones((num_objs,), dtype=torch.int64)`
 
to the following:

`label = torch.as_tensor(obj_ids, dtype=torch.int64)`

that will work if there are more than 2 types of object instances (`num_objs`) in the image. In fact, the new form will give the same exact vector of `labels = torch.ones((num_objs,), dtype=torch.int64)` and will work for any number of object instances. I have tried it with a dataset containing 60 object instances and all went well. 

Kind regards

**Comment by ver2king** [comment]
Regarding this issue, 
I only found a .html file when forking the repo that is relevant to the issue. Therefore, I safely assume that I shall fix that .html file?

**Comment by svekars** [comment]
This unfortunately won't work. We'll give half credit for the PR but it should not be merged.

**Comment by ver2king** [comment]
@svekars As you can see from the original issue (#960), that is the only issue that needs to be changed. May you request what needs to be adjusted to "work"?

**Comment by engichang1467** [comment]
@svekars and @morawi I'd like to know if this is still an unresolved issue. If so, then I'd like to work on it 😊

**Comment by morawi** [comment]
> @svekars and @morawi I'd like to know if this is still an unresolved issue. If so, then I'd like to work on it 😊
**From what I remember:**
I have tried this change with some other datasets, but not the PennFudan dataset. So, it is not working for the PennFudan for a reason. 
If @engichang1467 want to give it a try, I suggest trying it with some other datasets, then investigating why is it not working for the PennFudan, as outlined by @svekars.
